src/test/java/duke/TaskTest.java

```diff
@@ -0,0 +1,18 @@
+package duke;
+import static org.junit.jupiter.api.Assertions.assertEquals;
```

Comment:
Separation of import statements as per [guidelines](https://se-education.org/guides/conventions/java/intermediate.html#package-and-import-statements) 

```suggestion
package duke;

import static org.junit.jupiter.api.Assertions.assertEquals;
```